images/plbase/plbase-install.sh

```diff
@@ -29,7 +32,8 @@ yum -y install \
     git \
     graphviz \
     graphviz-devel \
-    libjpeg-devel # Needed by the Pillow package
+    libjpeg-devel \
+    gcc-c++
```

Comment:
One [hack](https://stackoverflow.com/a/12797512) for inline comments is like this:
```suggestion
    libjpeg-devel `# for the python Pillow package` \
    gcc-c++ `# for building the native bindings in packages/bind-mount`
```

Reply:
I'm going to land this as-is so I can make forward progress without waiting for another base image build in CI. We can tweak later if desired!

Reply:
Yeah, I'm fine either way -- inline is easier to keep up to date, but I worry that this is maybe a bit too devious 😄